api/handlers/container/create_test.go

```diff
@@ -1170,6 +1206,7 @@ func getDefaultCreateOpt(conf config.Config) types.ContainerCreateOptions {
 		BlkioDeviceWriteBps:  []string{},
 		BlkioDeviceReadIOps:  []string{},
 		BlkioDeviceWriteIOps: []string{},
+		Device:               []string{},
```

Comment:
nit: Devices

Reply:
This is actually from nerdctl container type - https://github.com/containerd/nerdctl/blob/694c405909e61736e7e117180409f403e54e170b/pkg/api/types/container_types.go#L154

Unfortunately the docker spec expects the input as "Devices" - https://docs.docker.com/reference/api/engine/version/v1.43/#tag/Container/operation/ContainerUpdate